.github/workflows/kind-cluster/docker-job-template.yaml

```diff
@@ -66,6 +67,8 @@ spec:
             value: "{{DOCKERFILE_PATH}}"
           - name: VERSION
             value: "{{VERSION}}"
+          - name: BUILD_TARGET
+            value: "{{BUILD_TARGET}}"
```

Comment:
add MODEL_NAME env var?

Reply:
Added the wrong env var. Removed in my latest commit.